static/js/tools/statsvar_menu.tsx

```diff
@@ -104,15 +106,21 @@ class Node extends Component<NodePropType, NodeStateType> {
     );
   };
   private _handleCheckboxClick = (): void => {
```

Comment:
please sync with @qlj-lijuan as the stats var menu code is in a constant change. A few things to note:

1) There maybe some logic specific to timeline chart now. This is a good example to see how we can make it general and let each app (timeline, choropleth, scatter) to extend it.

2) For the map view, I guess the statsvar menu should use radio button instead of check box, as you only want to select one stats var each time? @beets to confirm on the UX

Reply:
+1 to radio buttons